support-frontend/assets/components/subscriptionCheckouts/threeTierTerms.tsx

```diff
@@ -0,0 +1,70 @@
+import { css } from '@emotion/react';
+import { textSans } from '@guardian/source-foundations';
+import { FormSection } from 'components/checkoutForm/checkoutForm';
+import DirectDebitTerms from 'components/subscriptionCheckouts/directDebit/directDebitTerms';
+import type { PaymentMethod } from 'helpers/forms/paymentMethods';
+import { DirectDebit } from 'helpers/forms/paymentMethods';
+import type { Option } from 'helpers/types/option';
+
+const threeTierTerms = css`
+	${textSans.xxsmall()};
+	color: #606060;
+	p {
+		margin-top: 10px;
+	}
+`;
+
+export default function ThreeTierTerms(props: {
+	paymentMethod: Option<PaymentMethod>;
+	total: number;
+	currencySymbol: string;
+	paymentFrequency: 'month' | 'year';
+	discount?: {
+		total: number;
+		duration: number;
+		period: 'month' | 'year';
+	};
+}): JSX.Element {
+	const price = props.discount?.total ?? props.total;
+	return (
+		<>
+			<FormSection>
+				<div css={threeTierTerms}>
+					{props.paymentFrequency === 'month' && (
```

Comment:
Presume duplicated with yearly in case we have different payment frequency copies when legal come back with the finalised text